webapp/channels/src/components/mfa/setup/setup.tsx

```diff
@@ -137,7 +139,14 @@ export default class Setup extends React.PureComponent<Props, State> {
         let formClass = 'form-group';
         let errorContent;
         if (this.state.error) {
-            errorContent = <div className='form-group has-error'><label className='control-label'>{this.state.error}</label></div>;
+            errorContent = (
+                <div
+                    id='mfa-setup-error'
+                    className='form-group has-error'
+                >
```

Comment:
Could you also add `role='alert'` to this to have it get read out by the screen reader when it changes?
```suggestion
                <div
                    id='mfa-setup-error'
                    className='form-group has-error'
                    role='alert'
                >
```
Maria and I had a discussion a while back (https://hub.mattermost.com/private-core/pl/oemaj6qn33rb8nipqxgkyg4qpo) about how to write error messages, and I should go write up a tutorial on that since it's sure to come up in a lot of places